## Continuing here...

In [1]:
import pandas as pd
import plotly.express as px
import seaborn as sns
import numpy as np

GPU=pd.read_csv('GPU_Prices_And_Performance.csv',index_col=0)
GPU = GPU.drop(['NEWUSPrice','AVG1440pFPS','AVG4kFPS'],axis=1) #doing 1080p cuz that's what I play on, feel free to adjust
GPU

GPU  AVG1080pFPS  Newegg_New_Price  Ebay_Used_Price
1            RX 6900 XT        250.1             719.0           675.00
0              RTX 3090        246.5            1099.0           900.00
2           RTX 3080 Ti        241.8             889.0           814.95
3            RX 6800 XT        236.1             639.0           599.00
4              RTX 3080        232.4             739.0           650.00
..                  ...          ...               ...              ...
101  GTX 1050 Ti Mobile         33.4               NaN              NaN
105            GTX 1050         33.3               NaN              NaN
106              R9 270         32.8               NaN              NaN
108              RX 560         30.5               NaN              NaN
107             GTX 950         30.0               NaN              NaN

[109 rows x 4 columns]

In [2]:
GPU = GPU.sort_values('AVG1080pFPS',ascending=False)
GPU

GPU  AVG1080pFPS  Newegg_New_Price  Ebay_Used_Price
1            RX 6900 XT        250.1             719.0           675.00
0              RTX 3090        246.5            1099.0           900.00
2           RTX 3080 Ti        241.8             889.0           814.95
3            RX 6800 XT        236.1             639.0           599.00
4              RTX 3080        232.4             739.0           650.00
..                  ...          ...               ...              ...
101  GTX 1050 Ti Mobile         33.4               NaN              NaN
105            GTX 1050         33.3               NaN              NaN
106              R9 270         32.8               NaN              NaN
108              RX 560         30.5               NaN              NaN
107             GTX 950         30.0               NaN              NaN

[109 rows x 4 columns]

In [3]:
# Dollars per frame, used
GPU['Dollars_per_frame_used'] = GPU['Ebay_Used_Price'] / GPU['AVG1080pFPS']
# Dollars per frame, new
GPU['Dollars_per_frame_new'] = GPU['Newegg_New_Price'] / GPU['AVG1080pFPS']
GPU

GPU  AVG1080pFPS  Newegg_New_Price  Ebay_Used_Price  \
1            RX 6900 XT        250.1             719.0           675.00   
0              RTX 3090        246.5            1099.0           900.00   
2           RTX 3080 Ti        241.8             889.0           814.95   
3            RX 6800 XT        236.1             639.0           599.00   
4              RTX 3080        232.4             739.0           650.00   
..                  ...          ...               ...              ...   
101  GTX 1050 Ti Mobile         33.4               NaN              NaN   
105            GTX 1050         33.3               NaN              NaN   
106              R9 270         32.8               NaN              NaN   
108              RX 560         30.5               NaN              NaN   
107             GTX 950         30.0               NaN              NaN   

     Dollars_per_frame_used  Dollars_per_frame_new  
1                  2.698920               2.874850  
0                  3.651116               4.458418  
2                  3.370347               3.676592  
3                  2.537061               2.706480  
4                  2.796902               3.179862  
..                      ...                    ...  
101                     NaN                    NaN  
105                     NaN                    NaN  
106                     NaN                    NaN  
108                     NaN                    NaN  
107                     NaN                    NaN  

[109 rows x 6 columns]

In [4]:
pd.set_option('display.max_rows',200)
New_GPU = GPU[['GPU','AVG1080pFPS','Newegg_New_Price','Dollars_per_frame_new']]
New_GPU = New_GPU.sort_values('Dollars_per_frame_new',ascending=True)

Used_GPU = GPU[['GPU','AVG1080pFPS','Ebay_Used_Price','Dollars_per_frame_used']]
Used_GPU = Used_GPU.sort_values('Dollars_per_frame_used',ascending=True)

New_GPU

GPU  AVG1080pFPS  Newegg_New_Price  Dollars_per_frame_new
27             RTX 2060        107.8             249.0               2.309833
15           RX 6600 XT        135.8             319.0               2.349043
38       GTX 1660 SUPER         91.0             236.0               2.593407
40             GTX 1660         85.9             229.0               2.665891
60       GTX 1650 SUPER         73.2             196.0               2.677596
3            RX 6800 XT        236.1             639.0               2.706480
10           RX 6700 XT        163.2             449.0               2.751225
34          GTX 1660 Ti         96.3             266.0               2.762201
1            RX 6900 XT        250.1             719.0               2.874850
5               RX 6800        197.4             599.0               3.034448
20             RTX 3060        124.3             379.0               3.049075
8              RTX 3070        178.8             549.0               3.070470
11          RTX 3060 Ti        150.0             469.0               3.126667
4              RTX 3080        232.4             739.0               3.179862
6           RTX 3070 Ti        187.2             639.0               3.413462
24       RTX 2060 SUPER        114.7             399.0               3.478640
36             RTX 3050         92.9             324.0               3.487621
22             RTX 2070        120.8             438.0               3.625828
2           RTX 3080 Ti        241.8             889.0               3.676592
37             GTX 1070         93.6             399.0               4.262821
0              RTX 3090        246.5            1099.0               4.458418
9           RTX 2080 Ti        161.4             898.0               5.563817
12       RTX 2080 SUPER        145.5             892.0               6.130584
21           RX 5700 XT        122.8             799.0               6.506515
26           RX 5600 XT        105.8             698.0               6.597353
28           RX Vega 64        102.2             679.0               6.643836
25             GTX 1080        109.6             748.0               6.824818
29          GTX 1070 Ti        101.1             704.0               6.963403
16       RTX 2070 SUPER        128.3             898.0               6.999221
17          GTX 1080 Ti        129.2             934.0               7.229102
18             TITAN Xp        125.8             999.0               7.941176
14             RTX 2080        137.2            1097.0               7.995627
30           RX Vega 56         95.9             829.0               8.644421
23              RX 5700        112.7            1094.0               9.707187
13              TITAN V        139.9            2398.0              17.140815
7             TITAN RTX        166.3            3099.0              18.634997
19           Radeon VII        126.0            2389.0              18.960317
32          RTX 3050 Ti        102.3               NaN                    NaN
35             R9 295X2        101.1               NaN                    NaN
31          GTX TITAN X         97.9               NaN                    NaN
33      RTX 2080 Mobile         96.0               NaN                    NaN
39      GTX 1080 Mobile         93.1               NaN                    NaN
42      RTX 2070 Mobile         85.1               NaN                    NaN
46           GTX 980 Ti         84.5               NaN                    NaN
41       RTX 2080 Max-Q         82.3               NaN                    NaN
44       GTX 1080 Max-Q         82.2               NaN                    NaN
48               RX 590         80.7               NaN                    NaN
45      GTX 1070 Mobile         79.5               NaN                    NaN
47            R9 FURY X         78.0               NaN                    NaN
49      RTX 2060 Mobile         75.4               NaN                    NaN
65              HD 7990         73.8      

In [5]:
Used_GPU

GPU  AVG1080pFPS  Ebay_Used_Price  Dollars_per_frame_used
63               RX 480         59.8            75.00                1.254181
70              GTX 970         62.2            79.99                1.286013
75               R9 290         57.7            75.00                1.299827
53              GTX 980         73.7            99.99                1.356716
29          GTX 1070 Ti        101.1           139.99                1.384669
59               RX 580         71.4            99.99                1.400420
37             GTX 1070         93.6           139.99                1.495620
68               RX 570         59.6            89.99                1.509899
54       RX 5500 XT 8GB         73.5           115.00                1.564626
64         GTX 1060 3GB         63.7           101.00                1.585557
71              R9 290X         61.2           100.00                1.633987
47            R9 FURY X         78.0           129.00                1.653846
60       GTX 1650 SUPER         73.2           125.00                1.707650
40             GTX 1660         85.9           149.99                1.746100
38       GTX 1660 SUPER         91.0           160.00                1.758242
46           GTX 980 Ti         84.5           150.00                1.775148
61         GTX 1060 6GB         67.0           120.00                1.791045
25             GTX 1080        109.6           199.99                1.824726
27             RTX 2060        107.8           200.00                1.855288
26           RX 5600 XT        105.8           200.00                1.890359
34          GTX 1660 Ti         96.3           185.00                1.921080
72               R9 390         59.6           115.00                1.929530
48               RX 590         80.7           160.00                1.982652
35             R9 295X2        101.1           205.02                2.027893
15           RX 6600 XT        135.8           278.00                2.047128
94          GTX 1050 Ti         39.3            85.00                2.162850
17          GTX 1080 Ti        129.2           279.97                2.166950
24       RTX 2060 SUPER        114.7           255.00                2.223191
31          GTX TITAN X         97.9           219.00                2.236977
30           RX Vega 56         95.9           215.00                2.241919
69            GTX TITAN         67.3           151.99                2.258395
22             RTX 2070        120.8           275.00                2.276490
12       RTX 2080 SUPER        145.5           340.00                2.336770
16       RTX 2070 SUPER        128.3           300.00                2.338270
6           RTX 3070 Ti        187.2           450.00                2.403846
23              RX 5700        112.7           275.00                2.440106
10           RX 6700 XT        163.2           399.98                2.450858
14             RTX 2080        137.2           340.00                2.478134
8              RTX 3070        178.8           450.00                2.516779
3            RX 6800 XT        236.1           599.00                2.537061
5               RX 6800        197.4           504.44                2.555420
21           RX 5700 XT        122.8           319.00                2.597720
11          RTX 3060 Ti        150.0           399.99                2.666600
1            RX 6900 XT        250.1           675.00                2.698920
18             TITAN Xp        125.8           349.95                2.781797
9           RTX 2080 Ti        161.4           450.00                2.788104
4              RTX 3080        232.4           650.00                2.796902
36             RTX 3050         92.9           269.99                2.906243
20             RTX 3060        124.3           369.00                2.968624
28           RX Vega 64        102.2           313.49                3.067417
2           RTX 3080 Ti        241.8      

# In the used market, the 10 series still reigns supreme if you want to play near 100 fps.
### The 1070 ti is a particularly good deal right now. 8-8-2022

# In the new market, the 2060 and the 6600XT are kings, one for decent GTX 1080-like performance, and one for good performance in more modern games like Cyberpunk or Elden Ring.

### Personally, I would only go new if you're aiming for 6600XT performance or higher. The used market is currently full of insanely good deals. If you're aiming for between 60 - 110 fps, I'd go used, from a GTX 970 (will struggle in new games) to a GTX 1080 (will handle any new game at 1080p).

# Extra work for someone who wants 1440p info...

In [6]:
GPU=pd.read_csv('GPU_Prices_And_Performance.csv',index_col=0)
GPU = GPU.drop(['NEWUSPrice','AVG1080pFPS','AVG4kFPS'],axis=1) 
GPU

# Dollars per frame, used
GPU['Dollars_per_frame_used'] = GPU['Ebay_Used_Price'] / GPU['AVG1440pFPS']
# Dollars per frame, new
GPU['Dollars_per_frame_new'] = GPU['Newegg_New_Price'] / GPU['AVG1440pFPS']
GPU
New_GPU = GPU[['GPU','AVG1440pFPS','Newegg_New_Price','Dollars_per_frame_new']]
New_GPU = New_GPU.sort_values('Dollars_per_frame_new',ascending=True)

Used_GPU = GPU[['GPU','AVG1440pFPS','Ebay_Used_Price','Dollars_per_frame_used']]
Used_GPU = Used_GPU.sort_values('Dollars_per_frame_used',ascending=True)

New_GPU

GPU  AVG1440pFPS  Newegg_New_Price  Dollars_per_frame_new
15           RX 6600 XT        101.5             319.0               3.142857
27             RTX 2060         76.4             249.0               3.259162
38       GTX 1660 SUPER         67.2             236.0               3.511905
40             GTX 1660         63.5             229.0               3.606299
3            RX 6800 XT        176.4             639.0               3.622449
60       GTX 1650 SUPER         53.8             196.0               3.643123
10           RX 6700 XT        122.0             449.0               3.680328
34          GTX 1660 Ti         71.1             266.0               3.741210
1            RX 6900 XT        186.9             719.0               3.846977
5               RX 6800        147.6             599.0               4.058266
20             RTX 3060         92.0             379.0               4.119565
11          RTX 3060 Ti        113.8             469.0               4.121265
8              RTX 3070        133.1             549.0               4.124718
4              RTX 3080        173.0             739.0               4.271676
6           RTX 3070 Ti        139.3             639.0               4.587222
36             RTX 3050         68.5             324.0               4.729927
24       RTX 2060 SUPER         83.0             399.0               4.807229
2           RTX 3080 Ti        182.0             889.0               4.884615
22             RTX 2070         89.0             438.0               4.921348
37             GTX 1070         68.5             399.0               5.824818
0              RTX 3090        187.7            1099.0               5.855088
9           RTX 2080 Ti        123.6             898.0               7.265372
12       RTX 2080 SUPER        110.6             892.0               8.065099
21           RX 5700 XT         91.8             799.0               8.703704
28           RX Vega 64         76.5             679.0               8.875817
26           RX 5600 XT         78.6             698.0               8.880407
25             GTX 1080         80.8             748.0               9.257426
16       RTX 2070 SUPER         95.4             898.0               9.412998
29          GTX 1070 Ti         74.7             704.0               9.424364
17          GTX 1080 Ti         98.6             934.0               9.472617
18             TITAN Xp         94.9             999.0              10.526870
14             RTX 2080        103.2            1097.0              10.629845
30           RX Vega 56         71.7             829.0              11.562064
23              RX 5700         84.3            1094.0              12.977461
13              TITAN V        107.1            2398.0              22.390289
7             TITAN RTX        127.3            3099.0              24.344069
19           Radeon VII         94.2            2389.0              25.360934
32          RTX 3050 Ti         74.3               NaN                    NaN
35             R9 295X2         73.8               NaN                    NaN
31          GTX TITAN X         71.6               NaN                    NaN
33      RTX 2080 Mobile         72.3               NaN                    NaN
39      GTX 1080 Mobile         68.7               NaN                    NaN
42      RTX 2070 Mobile         62.4               NaN                    NaN
46           GTX 980 Ti         62.1               NaN                    NaN
41       RTX 2080 Max-Q         61.9               NaN                    NaN
44       GTX 1080 Max-Q         60.6               NaN                    NaN
48               RX 590         58.0               NaN                    NaN
45      GTX 1070 Mobile         58.2               NaN                    NaN
47            R9 FURY X         59.6               NaN                    NaN
49      RTX 2060 Mobile         53.5               NaN                    NaN
65              HD 7990         56.8      

In [7]:
Used_GPU

GPU  AVG1440pFPS  Ebay_Used_Price  Dollars_per_frame_used
63               RX 480         44.6            75.00                1.681614
75               R9 290         42.0            75.00                1.785714
70              GTX 970         44.6            79.99                1.793498
53              GTX 980         53.5            99.99                1.868972
29          GTX 1070 Ti         74.7           139.99                1.874029
59               RX 580         51.2            99.99                1.952930
37             GTX 1070         68.5           139.99                2.043650
68               RX 570         43.6            89.99                2.063991
47            R9 FURY X         59.6           129.00                2.164430
54       RX 5500 XT 8GB         52.8           115.00                2.178030
64         GTX 1060 3GB         46.0           101.00                2.195652
71              R9 290X         45.1           100.00                2.217295
60       GTX 1650 SUPER         53.8           125.00                2.323420
40             GTX 1660         63.5           149.99                2.362047
38       GTX 1660 SUPER         67.2           160.00                2.380952
46           GTX 980 Ti         62.1           150.00                2.415459
25             GTX 1080         80.8           199.99                2.475124
61         GTX 1060 6GB         48.4           120.00                2.479339
26           RX 5600 XT         78.6           200.00                2.544529
34          GTX 1660 Ti         71.1           185.00                2.601969
27             RTX 2060         76.4           200.00                2.617801
72               R9 390         43.7           115.00                2.631579
15           RX 6600 XT        101.5           278.00                2.738916
48               RX 590         58.0           160.00                2.758621
35             R9 295X2         73.8           205.02                2.778049
17          GTX 1080 Ti         98.6           279.97                2.839452
94          GTX 1050 Ti         28.8            85.00                2.951389
30           RX Vega 56         71.7           215.00                2.998605
31          GTX TITAN X         71.6           219.00                3.058659
24       RTX 2060 SUPER         83.0           255.00                3.072289
12       RTX 2080 SUPER        110.6           340.00                3.074141
22             RTX 2070         89.0           275.00                3.089888
69            GTX TITAN         48.7           151.99                3.120945
16       RTX 2070 SUPER         95.4           300.00                3.144654
6           RTX 3070 Ti        139.3           450.00                3.230438
23              RX 5700         84.3           275.00                3.262159
10           RX 6700 XT        122.0           399.98                3.278525
14             RTX 2080        103.2           340.00                3.294574
8              RTX 3070        133.1           450.00                3.380917
3            RX 6800 XT        176.4           599.00                3.395692
5               RX 6800        147.6           504.44                3.417615
21           RX 5700 XT         91.8           319.00                3.474946
11          RTX 3060 Ti        113.8           399.99                3.514851
1            RX 6900 XT        186.9           675.00                3.611557
9           RTX 2080 Ti        123.6           450.00                3.640777
18             TITAN Xp         94.9           349.95                3.687566
4              RTX 3080        173.0           650.00                3.757225
36             RTX 3050         68.5           269.99                3.941460
20             RTX 3060         92.0           369.00                4.010870
28           RX Vega 64         76.5           313.49                4.097908
2           RTX 3080 Ti        182.0      

# The lowest acceptable performance for 1440p IMO is found in either the 6700XT or the 3070, and although the 3070 performs better by about 11 frames per second, it will cost 100 USD higher.

### If you're going 1440p, you're gonna want a GPU that doesn't make you want to turn the details down (or revert to 1080p) to have your fps acceptably high.
### If you have a 1440p 60fps monitor, you can probably just aim for 90-100 fps, ie: 3060/6600XT range. Aiming for a card with a 60fps average is not a good idea, most games have areas where the card has to work much harder for extended periods of time. A card like the 1660 will drop to 30-40 often.

# I would say you should avoid the used market for an above 400 dollar investment, but the 3070 ti at 450 dollars used is a pretty good price, if you have the PSU to back it up. (290w just for the GPU alone)



# May as well do 4k...

In [8]:
GPU=pd.read_csv('GPU_Prices_And_Performance.csv',index_col=0)
GPU = GPU.drop(['NEWUSPrice','AVG1080pFPS','AVG1440pFPS'],axis=1) 
GPU

# Dollars per frame, used
GPU['Dollars_per_frame_used'] = GPU['Ebay_Used_Price'] / GPU['AVG4kFPS']
# Dollars per frame, new
GPU['Dollars_per_frame_new'] = GPU['Newegg_New_Price'] / GPU['AVG4kFPS']
GPU
New_GPU = GPU[['GPU','AVG4kFPS','Newegg_New_Price','Dollars_per_frame_new']]
New_GPU = New_GPU.sort_values('Dollars_per_frame_new',ascending=True)

Used_GPU = GPU[['GPU','AVG4kFPS','Ebay_Used_Price','Dollars_per_frame_used']]
Used_GPU = Used_GPU.sort_values('Dollars_per_frame_used',ascending=True)

New_GPU

GPU  AVG4kFPS  Newegg_New_Price  Dollars_per_frame_new
15           RX 6600 XT      59.5             319.0               5.361345
27             RTX 2060      45.3             249.0               5.496689
38       GTX 1660 SUPER      40.3             236.0               5.856079
40             GTX 1660      38.0             229.0               6.026316
60       GTX 1650 SUPER      32.1             196.0               6.105919
34          GTX 1660 Ti      42.5             266.0               6.258824
10           RX 6700 XT      70.6             449.0               6.359773
3            RX 6800 XT     100.1             639.0               6.383616
11          RTX 3060 Ti      69.3             469.0               6.767677
1            RX 6900 XT     105.9             719.0               6.789424
8              RTX 3070      80.8             549.0               6.794554
20             RTX 3060      55.4             379.0               6.841155
4              RTX 3080     105.0             739.0               7.038095
6           RTX 3070 Ti      89.0             639.0               7.179775
5               RX 6800      79.7             599.0               7.515684
36             RTX 3050      41.1             324.0               7.883212
2           RTX 3080 Ti     112.6             889.0               7.895204
24       RTX 2060 SUPER      50.0             399.0               7.980000
22             RTX 2070      53.7             438.0               8.156425
0              RTX 3090     117.9            1099.0               9.321459
37             GTX 1070      40.6             399.0               9.827586
9           RTX 2080 Ti      75.6             898.0              11.878307
12       RTX 2080 SUPER      67.2             892.0              13.273810
21           RX 5700 XT      54.8             799.0              14.580292
28           RX Vega 64      45.6             679.0              14.890351
26           RX 5600 XT      46.8             698.0              14.914530
25             GTX 1080      48.2             748.0              15.518672
16       RTX 2070 SUPER      57.6             898.0              15.590278
17          GTX 1080 Ti      59.7             934.0              15.644891
29          GTX 1070 Ti      44.6             704.0              15.784753
18             TITAN Xp      58.9             999.0              16.960951
14             RTX 2080      62.3            1097.0              17.608347
30           RX Vega 56      42.8             829.0              19.369159
23              RX 5700      50.3            1094.0              21.749503
13              TITAN V      66.7            2398.0              35.952024
7             TITAN RTX      77.8            3099.0              39.832905
19           Radeon VII      56.3            2389.0              42.433393
32          RTX 3050 Ti      44.3               NaN                    NaN
35             R9 295X2      46.7               NaN                    NaN
31          GTX TITAN X      43.0               NaN                    NaN
33      RTX 2080 Mobile      43.6               NaN                    NaN
39      GTX 1080 Mobile      41.0               NaN                    NaN
42      RTX 2070 Mobile      38.2               NaN                    NaN
46           GTX 980 Ti      37.0               NaN                    NaN
41       RTX 2080 Max-Q      37.4               NaN                    NaN
44       GTX 1080 Max-Q      36.2               NaN                    NaN
48               RX 590      33.8               NaN                    NaN
45      GTX 1070 Mobile      34.5               NaN                    NaN
47            R9 FURY X      36.3               NaN                    NaN
49      RTX 2060 Mobile      31.7               NaN                    NaN
65              HD 7990      32.5               NaN                    NaN
53              GTX 980      32.0               NaN                    NaN
54       RX 5500 XT 8GB      30.8            

In [9]:
Used_GPU

GPU  AVG4kFPS  Ebay_Used_Price  Dollars_per_frame_used
63               RX 480      27.4            75.00                2.737226
75               R9 290      26.2            75.00                2.862595
70              GTX 970      27.4            79.99                2.919343
53              GTX 980      32.0            99.99                3.124687
29          GTX 1070 Ti      44.6           139.99                3.138789
59               RX 580      29.7            99.99                3.366667
37             GTX 1070      40.6           139.99                3.448030
68               RX 570      25.6            89.99                3.515234
47            R9 FURY X      36.3           129.00                3.553719
71              R9 290X      27.9           100.00                3.584229
64         GTX 1060 3GB      27.4           101.00                3.686131
54       RX 5500 XT 8GB      30.8           115.00                3.733766
60       GTX 1650 SUPER      32.1           125.00                3.894081
40             GTX 1660      38.0           149.99                3.947105
38       GTX 1660 SUPER      40.3           160.00                3.970223
46           GTX 980 Ti      37.0           150.00                4.054054
25             GTX 1080      48.2           199.99                4.149170
61         GTX 1060 6GB      28.8           120.00                4.166667
26           RX 5600 XT      46.8           200.00                4.273504
34          GTX 1660 Ti      42.5           185.00                4.352941
35             R9 295X2      46.7           205.02                4.390150
27             RTX 2060      45.3           200.00                4.415011
72               R9 390      25.1           115.00                4.581673
15           RX 6600 XT      59.5           278.00                4.672269
17          GTX 1080 Ti      59.7           279.97                4.689615
48               RX 590      33.8           160.00                4.733728
94          GTX 1050 Ti      17.1            85.00                4.970760
30           RX Vega 56      42.8           215.00                5.023364
6           RTX 3070 Ti      89.0           450.00                5.056180
12       RTX 2080 SUPER      67.2           340.00                5.059524
31          GTX TITAN X      43.0           219.00                5.093023
24       RTX 2060 SUPER      50.0           255.00                5.100000
22             RTX 2070      53.7           275.00                5.121043
16       RTX 2070 SUPER      57.6           300.00                5.208333
14             RTX 2080      62.3           340.00                5.457464
23              RX 5700      50.3           275.00                5.467197
8              RTX 3070      80.8           450.00                5.569307
69            GTX TITAN      27.1           151.99                5.608487
10           RX 6700 XT      70.6           399.98                5.665439
11          RTX 3060 Ti      69.3           399.99                5.771861
21           RX 5700 XT      54.8           319.00                5.821168
18             TITAN Xp      58.9           349.95                5.941426
9           RTX 2080 Ti      75.6           450.00                5.952381
3            RX 6800 XT     100.1           599.00                5.984016
4              RTX 3080     105.0           650.00                6.190476
5               RX 6800      79.7           504.44                6.329235
1            RX 6900 XT     105.9           675.00                6.373938
36             RTX 3050      41.1           269.99                6.569100
20             RTX 3060      55.4           369.00                6.660650
28           RX Vega 64      45.6           313.49                6.874781
2           RTX 3080 Ti     112.6           814.95                7.237567
0              RTX 3090     117.9           900.00                7.633588
7             TITAN RTX      77.8           8

# Honestly, it sounds crazy, but the 6900XT, 3080ti, and 3090 are the only ones I would consider for 4k, if I was really serious about 4k.
# Anything lower, and I'd just end up playing in 1440p. I prefer FPS to fidelity, but if you only play cinematic games and want to stay at around 60 fps, 3070 ti might be able to do it. It's cutting it close though, you may see dips below 60.